doc/api/next_api_changes/removals/26962-IA.rst

```diff
@@ -0,0 +1,22 @@
+Removal change template
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Deprecated Classes Removed
+##########################
+
+The following deprecated classes from version 3.7 have been removed:
+
+- `matplotlib.backends.backend_ps.PsBackendHelper`
+- `matplotlib.backends.backend_webagg.ServerThread`
```

Comment:
```suggestion
- ``matplotlib.backends.backend_ps.PsBackendHelper``
- ``matplotlib.backends.backend_webagg.ServerThread``
```

Since these are removed, they no longer have a link, so they need to be code style (double backtick) instead of references (single backtick)